### DISCUSSION TITLE
Welcome to ductwork Discussions!

### DISCUSSION THREAD

**Discussion by Tokarzewski** [ModelTok/wenta#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ianvg** [comment]
Hello, @Tokarzewski I am interested in the kind of work you have been doing! I recently forked your code and created a very basic GUI. I am a building engineer in the U.S., so I am I interested in expanding your work in imperial units and fittings. 

I am really most interested in creating a replace for the Trane Trace Ductulator. I recently heard from customer support that they are no longer supporting that tool into the future, so I am interested in developing a working replacement for that one tool. I would like to develop some sort of database that others can contribute data too (after being vetted), so that results from current research, papers, and studies can be utilized in this tool.

My fork is located here: https://github.com/ianvg/pyduct/tree/main. 

I am really new to programming, and to be honest I am mainly using ChapGPT for development of this tool (free time is not one my big luxuries!). But I would like to learn good principles of Python along the way.

Hopefully, some of what I do, can be useful for you!

Thanks,
Ian

**Reply by Tokarzewski** [reply]
Hi @ianvg 

Sounds good, can you provide the link to the "Trane Trace Ductulator" tool you are trying to replace? 

Regards,
Bart
